src/app/_domain/constants/index.ts

```diff
@@ -1,2 +1,3 @@
 export const USERS_KEY = "users";
 export const PROFESSIONS_KEY = "professions";
+export const APPLICATIONS_KEY = "applications";
```

Comment:
I would consider having these keys as enums. If they are of the same nature as `APPLICATIONS_KEY`, which is a query key for Tanstack, I would suggest encapsulating these keys as such:
```suggestion
export enum TanstackQueryKeys = {
   applications = 'applications'
}
```